website/src/contribute/FAQ.md

```diff
@@ -0,0 +1,84 @@
+---
+layout: section
+section_menu: section-menu/contribute.html
+title: "Beam Contribution FAQ"
+permalink: /contribute/FAQ/
+---
+<!--
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+## Apache Beam Contribution FAQ
+
+Frequently asked questions about contributing to Apache Beam.
```

Comment:
Missing close tag makes this render as an empty page: http://apache-beam-website-pull-requests.storage.googleapis.com/6857/contribute/FAQ/index.html

Reply:
Fixed. Also added a TODO to move this to the wiki/

Reply:
@alanmyrvold you now have the wiki access